job-service-postgres-container/pom.xml

```diff
@@ -0,0 +1,254 @@
+                            <run>
+                                <ports>
+                                    <port>${postgres.db.port}:5432</port>
+                                </ports>
+                                <env>
+                                    <POSTGRES_PASSWORD>${js.database.password}</POSTGRES_PASSWORD>
+                                    <POSTGRES_USER>${js.database.username}</POSTGRES_USER>
+                                    <POSTGRES_DB>${js.database.name}</POSTGRES_DB>
+                                </env>
+                                <wait>
+                                    <log>PostgreSQL init process complete</log>
+                                    <time>120000</time>
+                                    <shutdown>500</shutdown>
+                                </wait>
+                                <log>
+                                    <enabled>true</enabled>
+                                </log>
+                            </run>
+                        </image>
+                        
+                        <!-- Run old version of jobservice postgres container against postgres database to install jobservice schema. -->
+                        <image>
+                            <alias>job-service-postgres-old</alias>
+                            <name>jobservice/job-service-postgres:2.5</name>
+                            <run>
+                                <links>
+                                    <link>${js.database.host}</link>
+                                </links>
+                                <env>
+                                    <POSTGRES_DB>${js.database.name}</POSTGRES_DB>
+                                </env>
+                                <cmd>
+                                    ./install_job_service_db.sh -fd -db.connection jdbc:postgresql://${js.database.host}:5432 -db.name ${js.database.name} -db.user ${js.database.username} -db.pass ${js.database.password}
+                                </cmd>
+                                <wait>
+                                    <log>DB update finished.</log>
+                                    <time>120000</time>
+                                    <shutdown>500</shutdown>
+                                </wait>
+                                <log>
+                                    <enabled>true</enabled>
+                                </log>
+                            </run>
+                        </image>
+                        
+                        <!-- Run jobservice postgres container against postgres database to upgrade jobservice schema. -->
+                        <image>
+                            <alias>job-service-postgres</alias>
+                            <name>${dockerJobServiceOrg}job-service-postgres${dockerProjectVersion}</name>
+                            <build>
+                                <from>cafapi/java-postgres:1.0</from>
```

Comment:
1-part

Reply:
1 part version not available: https://hub.docker.com/r/cafapi/java-postgres/tags